src/App.jsx

```diff
@@ -2,18 +2,47 @@ import { useState } from 'react'
 import './App.css'
 
 function App() {
-  const [count, setCount] = useState(0)
   const [name, setName] = useState("");
-  const [day, setDay] = useState(0)
+  const [day, setDay] = useState(1)
   const [month, setMonth] = useState('')
+  const [daysToGo, setDaysToGo] = useState(0)
+  const [isNotSubmitted, setIsNotSubmitted] = useState(true)
 
+  const dayMin = 1
+  const dayMax = 31
 
+  const today = new Date()
+  let birthday;
+
+  const updateDay = event => {
+    const birthdayDay = Math.max(dayMin, Math.min(dayMax, Number(event.target.value)))
+    setDay(birthdayDay)
+  }
+
+  const submit = () => {
+    setIsNotSubmitted(false)
+    birthday = new Date(`${month.toString()}-${day.toString()}`)
+    let daysDifference = Math.ceil((birthday.getTime() - today.getTime()) / (1000 * 3600 * 24)).toFixed(0)
+    console.log(daysDifference)
+    while (daysDifference < -367) {
+      daysDifference = parseInt(daysDifference) + 365
+    }
+    if (daysDifference >= 0) {
+      setDaysToGo(daysDifference)
+    } else {
+        setDaysToGo(parseInt(daysDifference) + 365)
+      }
+
+  }
 
   return (
     <>
       
-      <h1>Hello There!</h1>
+    {isNotSubmitted ? (
```

Comment:
Rather than having this massive ternary I would lean towards having a separate component that has 
if(isSubmitted){ // Show form} 

return {// Display birthday message}